src/core/Microsoft.PowerToys.Settings.UI/Views/PowerPreviewPage.xaml

```diff
@@ -43,25 +48,41 @@
 
         <StackPanel Orientation="Vertical" 
                     x:Name="PowerPreviewView">
+            <TextBlock x:Uid="FileExplorerPreview_RunAsAdminRequired"
+                       Foreground="{ThemeResource SystemControlErrorTextForegroundBrush}"
+                       Visibility="{Binding Mode=OneWay, Path=IsElevated, Converter={StaticResource BoolToVisibilityConverter}}"
+                       Margin="{StaticResource SmallBottomMargin}"/>
+
+            <TextBlock x:Uid="FileExplorerPreview_AffectsAllUsers"
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.IsElevated, Converter={StaticResource ModuleEnabledToForegroundConverter}}"
+                       Margin="{StaticResource SmallBottomMargin}"/>
+
             <TextBlock x:Uid="FileExplorerPreview_PreviewPane_GroupSettings"
                        Style="{StaticResource SettingsGroupTitleStyleAsHeader}"
-                       Foreground="{x:Bind Mode=OneWay, Path=IsEnabled, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.IsElevated, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
 
             <ToggleSwitch x:Uid="FileExplorerPreview_ToggleSwitch_Preview_SVG"
                           Margin="{StaticResource SmallTopMargin}"
-                          IsOn="{Binding Mode=TwoWay, Path=SVGRenderIsEnabled}" />
+                          IsOn="{Binding Mode=TwoWay, Path=SVGRenderIsEnabled}"                          
+                          IsEnabled="{Binding Mode=OneWay, Path=IsElevated}"/>
 
             <ToggleSwitch x:Uid="FileExplorerPreview_ToggleSwitch_Preview_MD"
                           Margin="{StaticResource SmallTopMargin}"
-                          IsOn="{Binding Mode=TwoWay, Path=MDRenderIsEnabled}" />
-            
+                          IsOn="{Binding Mode=TwoWay, Path=MDRenderIsEnabled}"
+                          IsEnabled="{Binding Mode=OneWay, Path=IsElevated}"/>
+
             <TextBlock x:Uid="FileExplorerPreview_IconThumbnail_GroupSettings"
                        Style="{StaticResource SettingsGroupTitleStyle}"
-                       Foreground="{x:Bind Mode=OneWay, Path=IsEnabled, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
-            
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.IsElevated, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
+
+            <TextBlock x:Uid="FileExplorerPreview_RebootRequired"
```

Comment:
Because it is a hint/warning I would paint it red too:

```suggestion
            <TextBlock x:Uid="FileExplorerPreview_RebootRequired"
            Foreground="{ThemeResource SystemControlErrorTextForegroundBrush}"
```

Reply:
Red feels more like error/blocking text to me. @crutas @niels9001 thoughts here? We want it to warn the user but we don't want it to indicate that they can't do something (like we do for run as admin permissions). Would making the text bold work?

Reply:
I'd do this as a warning yellow versus error red.

Reply:
Yellow as warning colour isn't working because the official ErrorTextBrush is using yellow in dark mode (instead of red in light mode).

I think making text bold could be ab idea or we use the exclamation (⚠️) sign in front of the text. But bold would be my favourite.

Reply:
@htcfreek I've made both the warning SemiBold. I tried Bold but it seemed like too much since the headers are SemiBold. I've updated the screenshots in the PR description. I did not add the exclamation sign as I wasn't really sure if that would be consistent with Windows Settings.
@crutkas I don't think yellow text on light theme would be accessible, so I've gone ahead with semi-bold for the PR. I think if needed it could be tweaked in a separate PR.